package.json

```diff
@@ -23,38 +23,57 @@
     "type": "git",
     "url": "https://github.com/ioBroker/ioBroker.node-red"
   },
+  "engines": {
+    "node": ">=16"
```

Comment:
Formally needs to be >=16.4.0 because of the "node: "namespaced packages you use

Reply:
Thanks